src/vma/lwip/tcp.c

```diff
@@ -82,6 +82,13 @@ void register_tcp_seg_free(tcp_seg_free_fn fn)
 }
 #endif
 
+tcp_set_error_staus_fn external_tcp_set_error_status;
+
+void register_tcp_set_error_status(tcp_set_error_staus_fn fn)
+{
+	external_tcp_set_error_status = fn;
+}
+
```

Comment:
Consider moving m_error_status into the pcb struct.
This way you will not need to add this callback.

Reply:
I agree. The code will be updated.